ui/src/app/views/page-template-signed-out/component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+  selector: 'app-page-template-signed-out',
+  templateUrl: './component.html',
+  styleUrls: ['./component.css']
+})
+export class PageTemplateSignedOutComponent {
+  @Input() imageSrc: string;
+  headerImg: string;
+
+  constructor() {
+    this.imageSrc = '';
```

Comment:
If a class property needs initialization in typescript, it may be placed directly on the property, e.g.

```
@Input() imageSrc: string = '';
headerImg: string = '/assets/etc/etc/';
```

And then you could leave out the constructor (an empty constructor isn't needed either).

Reply:
Will take care of this